app/services/expired_license_allower.rb

```diff
@@ -0,0 +1,64 @@
+# Needs the "raw" response error codes, which needs to happen before translation
+class ExpiredLicenseAllower
+  # @param IdentityDocAuth::Response
+  def initialize(response)
+    @response = response
+  end
+
+  # When we allow expired drivers licenses, returns a successful response
+  # @return [IdentityDocAuth::Response]
+  def processed_response
+    if document_expired?
+      if IdentityConfig.store.proofing_allow_expired_license && allowable_expired_license?
+        IdentityDocAuth::Response.new(
+          success: true, # new response with explicit true overrides success
+          errors: {},
+          exception: response.exception,
+          extra: response.extra.merge(
+            document_expired: document_expired?,
+            reproof_at: reproof_at,
+          ),
+          pii_from_doc: response.pii_from_doc,
+        )
+      else
+        response.merge(
+          IdentityDocAuth::Response.new(
+            success: true, # merge uses "&&" to combine success, so this does not override
+            extra: {
+              document_expired: document_expired?,
+              would_have_passed: allowable_expired_license?,
```

Comment:
High level flow step 1b: when feature flag is disabled, we add these two keys instead